clippy_lints/src/shadow.rs

```diff
@@ -42,9 +43,10 @@ declare_lint! {
 /// let x = x + 1;
 /// ```
 declare_lint! {
-    pub SHADOW_REUSE, Allow,
+    pub SHADOW_REUSE,
+    Allow,
     "rebinding a name to an expression that re-uses the original value, e.g. \
-    `let x = x + 1`"
+     `let x = x + 1`"
```

Comment:
That's just what Emacs' rust-mode does when autoindenting. (Personally I also think it makes sense to indent like that.)